docker/Makefile

```diff
@@ -1,13 +1,12 @@
-LT = $(shell cat ~/.lotus/token)
-#LT = $(shell docker exec lotustest cat /root/.lotus/token)
-
-fresh: down up
-.PHONY: fresh-up
-
 down:
-	@TEXFILTOOLS_LOTUSTOKEN=$(LT) docker-compose down
+	docker-compose down
+	docker-compose -p embed down
 .PHONY: down
 
-up: 
-	TEXFILTOOLS_LOTUSTOKEN=$(LT) docker-compose up --build 
-.PHONY: up
\ No newline at end of file
+fresh: down
```

Comment:
Simplification of makefile. All env variables are defined in the _docker-compose_ files.